app/controllers/kudos_controller.rb

```diff
@@ -4,37 +4,29 @@ class KudosController < ApplicationController
   before_action :set_kudo, only: %i[show edit update destroy]
```

Comment:
[suggestion] Since you cleaned up Admin Controller so well, maybe it is a time to refactor this controller, and it's callback as well :)

Reply:
Fixed.